### DISCUSSION TITLE
Vercel Edge function bandwidth pricing

### DISCUSSION THREAD

**Discussion by tasgon** [vercel/community#3984] [Feature Question] [answered]
### Question

How does Vercel charge for egress from Edge functions? Is it the same as serverless functions?

**Comment by swarnava** [comment, marked as answer]
Hi,

You can review Edge function pricing here: https://vercel.com/docs/functions/edge-functions/usage-and-pricing#pricing

**Reply by gothy** [reply]
@swarnava I'm sorry, but I don't see anything bandwidth-related on that page... Maybe it has changed since Sep 9? 🤔

**Reply by gothy** [reply]
Ah, nevermind, i found my answer here https://vercel.com/docs/limits/usage#bandwidth
